sdk/core/azure-core/examples/examples_async.py

```diff
@@ -73,7 +73,7 @@
 
     # It can also be overridden per operation.
     async with AsyncPipelineClient(base_url=url, config=config) as client:
-        response = await client._pipeline.run(request, redirect_max=5)
+        response = await client._pipeline.run(request, permit_redirects=True, redirect_max=5)
```

Comment:
permit_redirects=True is the default value, not right?

Reply:
Yes it is default value. The reason I used it here is because a couple lines up I disable redirects and then wanted to re-enable it for the test example. I'll move it to a separate test.